boxoffice/views/forms.py

```diff
@@ -0,0 +1,32 @@
+from wtforms import Form
```

Comment:
You can use the `MyForm(csrf_enabled=False)` constructor to turn it off. Baseframe's Form has other useful features including logging that we can't lose here.